gitman/cli.py

```diff
@@ -294,7 +302,8 @@ def _run_command(function, args, kwargs):
     except exceptions.UncommittedChanges as exception:
         _show_error(exception)
         exit_message = (
-            "Run again with '--force' to discard changes "
+            "Run again with '--force' resp. '--force-interactive' to "
+            "discard changes "
```

Comment:
Should `resp.` be `or`?

Reply:
"respectively", "alternatively", "or" have in my mind all the same meaning.
Because "or" was already used before the `--skip-changes` option I was thinking how to avoid another "or" in that sentence. More a stylistic topic. Do you think another "or" is better?

Reply:
How about `"Run again with --force/--force-interactive to discard changes"`?